brainstormforce/ultimate-addons-for-gutenberg
# Fix: Multi-button - Parent typography not working

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/1be82449-dcfc-44be-8229-92d6f86fde85?t=attributes&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop multi-buttons block
- Select typography Crafty girl and check is it applied in the editor as well as on the front-end.

### Checklist:
